.env.example

```
@@ -44,6 +44,9 @@ NUXT_BOOKS_ENRICHMENT_BATCH_SIZE=20
 NUXT_BOOKS_ENRICHMENT_CONCURRENCY=4
 NUXT_BOOKS_ENRICHMENT_LEASE_SECONDS=900
 NUXT_BOOKS_ENRICHMENT_MAX_ATTEMPTS=5
+# Reserve metadata, cover-download, and persistence time before starting a recovery attempt.
+NUXT_BOOKS_ENRICHMENT_RECOVERY_TIME_BUDGET_SECONDS=840
+NUXT_BOOKS_ENRICHMENT_RECOVERY_SAFETY_SECONDS=35
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NUXT_BOOKS_ENRICHMENT_RECOVERY_SAFETY_SECONDS key should go before the NUXT_BOOKS_ENRICHMENT_RECOVERY_TIME_BUDGET_SECONDS key

(UnorderedKey)